contracts/BatchReveal.sol

```diff
@@ -432,7 +541,7 @@ abstract contract BatchReveal is
     }
 
     /// @dev Force reveal, should be restricted to owner
-    function _forceReveal() internal batchRevealEnabled {
+    function forceReveal() external override onlyOwner batchRevealEnabled {
```

Comment:
does this mean that we must transfer ownership of this contract as well when we transfer ownership of the Launchpeg contract?

Reply:
yeah, whoever deploys the contract will own it, I believe the same multisig that deploys launchpeg contracts

https://github.com/traderjoe-xyz/launchpeg/blob/46192c8a4024d37dcd47f0908434dabc229d8be6/contracts/LaunchpegFactory.sol#L245-L271